owasp-top10-2017-apps/a9/cimentech/README.md

```diff
@@ -1,43 +1,132 @@
 # Cimentech
- > This is a simple web application built with Drupal that contains an example of a component with a known vulnerabilty.
 
 <p align="center">
- <img width="560" height="" src="docs/attack1.png">
+    <img src="images/attack1.png"/>
 </p>
 
-## What does it mean to use a component with known vulnerabilities?
+Cimentech is a simple web application built with Drupal that contains an example of a component with a known vulnerabilty and it's main goal is to demonstrate how an attacker could exploit it.
+
+## Index
+
+- [Definition](#what-does-it-mean-to-use-a-component-with-known-vulnerabilities)
+- [Setup](#setup)
+- [Attack narrative](#attack-narrative)
+- [Objectives](#secure-this-app)
+- [Solutions](#pr-solutions)
+- [Contributing](#contributing)
 
-Definition from [OWASP](https://www.owasp.org/images/7/72/OWASP_Top_10-2017_%28en%29.pdf.pdf):
+## What does it mean to use a component with known vulnerabilities?
 
 Components, such as libraries, frameworks, and other software modules, run with the same privileges as the application. If a vulnerable component is exploited, such an attack can facilitate serious data loss or server takeover. Applications and APIs using components with known vulnerabilities may undermine application defenses and enable various attacks and impacts.
```

Comment:
What do you think of adding the following at the beginning of this paragraph: `Imagine that components, such as ....`

Reply:
Thanks @rafaveira3! It does look better with what you proposed!